src/sentry/static/sentry/app/components/group/externalIssueForm.tsx

```diff
@@ -71,17 +91,20 @@ class ExternalIssueForm extends AsyncComponent {
     });
   };
 
-  getDynamicFields(integrationDetails) {
+  getDynamicFields(integrationDetails?: IntegrationIssueConfig) {
     integrationDetails = integrationDetails || this.state.integrationDetails;
     const {action} = this.props;
-    const config = integrationDetails[`${action}IssueConfig`];
+    const config: IssueConfigField[] = integrationDetails[`${action}IssueConfig`];
 
     return config
-      .filter(field => field.updatesForm)
-      .reduce((a, field) => ({...a, [field.name]: field.default}), {});
+      .filter((field: IssueConfigField) => field.updatesForm)
+      .reduce(
+        (a: object, field: IssueConfigField) => ({...a, [field.name]: field.default}),
```

Comment:
This is old code, but can we maybe update to use `Object.fromEntries()` to create the object?

Reply:
@billyvg hmmm...not sure that works here since I'm not mapping a list of key-value pairs. But there are definitely other places in this file that could use that function.

Reply:
Yeah it would you to map after the filter to return a list of `[name, default]`

Reply:
@billyvg I see what you mean. Updated